phcovid/phcovid.py

```diff
@@ -70,6 +76,8 @@ def get_cases(
     df_aliased[["contacts", "num_contacts"]] = extract_contact_info(
         df_aliased.travel_history
     )
+    df_aliased["confirmation_date"] = df_aliased["confirmation_date"].apply(lambda x: fix_dates(x))
```

Comment:
Can you generalize this by applying it to all the "date columns" via for loop? Can define date columns as list, the loop through this.

Something like

```
DATE_COLS = ["contacts", "num_contacts"]
...
# in get_cases
for col in DATE_COLS:
    df_aliased["confirmation_date"] = df_aliased["confirmation_date"].apply(lambda x: fix_dates(x))

```